staging/src/k8s.io/apiserver/pkg/server/options/priority_and_fairness.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"k8s.io/apiserver/pkg/server"
+	utilflowcontrol "k8s.io/apiserver/pkg/util/flowcontrol"
+	"k8s.io/utils/path"
+)
+
+// PriorityAndFairnessOptions holds the priority and fairness options.
+type PriorityAndFairnessOptions struct {
+	// ConfigFile is the file path with flow control configuration
+	ConfigFile string
+}
+
+// NewPriorityAndFairnessOptions creates a new instance of PriorityAndFairnessOptions.
+//
+// The option is to point to a configuration file for priority and fairness parameters.
+func NewPriorityAndFairnessOptions() *PriorityAndFairnessOptions {
+	return &PriorityAndFairnessOptions{}
+}
+
+// AddFlags adds flags related to priority and fairness
+// for a specific APIServer to the specified FlagSet.
+func (o *PriorityAndFairnessOptions) AddFlags(fs *pflag.FlagSet) {
+	if o == nil {
+		return
+	}
+
+	fs.StringVar(&o.ConfigFile, "flow-control-config-file", o.ConfigFile,
```

Comment:
The name issue appears here too.

Reply:
Did not spot that during the rename. Changed.